sunpy/net/dataretriever/sources/gong.py

```diff
@@ -140,7 +149,7 @@ def download_from_nso(id, ObsID, time_range):
             elif (pattern_ == url_pattern_2):
                 urls = [url_pattern_2.format(ObsID=id[0].upper()) for id in instruments_to]
                 arr = [Scraper(pattern__).filelist(timerange) for pattern__ in urls]
-                result.extend([url for url in arr if len(url)>0])
+                [result.extend(url) for url in arr if len(url)>0]
```

Comment:
I don't really fancy this...  you'll have to convince me why we use this  this way.